src/main/java/com/greenfox/lvlup/LvlupApplication.java

```diff
@@ -9,6 +9,15 @@
 public class LvlupApplication {
 
   public static void main(String[] args) {
+    String s;
```

Comment:
These lines are not necessary for our application, and I think are left here because of trying out things

Reply:
Yes, this was for testing sonarqube, I just removed.